tutorials/io/plot_30_reading_fnirs_data.py

```diff
@@ -37,6 +37,35 @@
              stored in the .snirf format.
 
 
+.. _import-boxy:
+
+BOXY (.txt)
+===========
+
+BOXY recordings can be read in using :func:`mne.io.read_raw_boxy`.
+The BOXY software and ISS Imagent I and II devices are frequency domain
+systems that store data in a single ``.txt`` file containing what they call
+(with MNE's name for that type of data in parens):
+
+- DC
+    All light collected by the detector (``fnirs_cw_amplitude``)
+- AC
+    High-frequency modulated light intensity (``fnirs_fd_ac_amplitude``)
+- Phase
+    Information for each source and detector (``fnirs_fd_phase``)
```

Comment:
This doesnt make sense to me, isnt the DC and AC also for each source and detector? I assume this should be something like ... high frequency modulated light phase (delay? relative to emmited AC signal?)

Reply:
Yes agreed, "Phase of the modulated light"